docs/servo_power_monitor/src/ui.ts

```diff
@@ -16,8 +16,12 @@ const analyzeButton = document.getElementById(
 const selectDutSerialButton = document.getElementById(
   'selectDutSerialButton'
 ) as HTMLButtonElement;
-const form = document.getElementById('form') as HTMLFormElement;
-const input = document.getElementById('input') as HTMLInputElement;
+const DutCommandForm = document.getElementById(
+  'DutCommandForm'
+) as HTMLFormElement;
+const DutCommandInput = document.getElementById(
+  'DutCommandInput'
+) as HTMLInputElement;
```

Comment:
they should be lowerCamelCase, not UpperCamelCase
https://google.github.io/styleguide/tsguide.html#naming-rules-by-identifier-type